manager/controllers/app/plotter_controller.go

```diff
@@ -0,0 +1,246 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package app
+
+import (
+	"context"
+	"emperror.dev/errors"
+	"github.com/go-logr/logr"
+	app "github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"github.com/ibm/the-mesh-for-data/pkg/multicluster"
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"os"
+	"reflect"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	ctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"time"
+)
+
+// PlotterReconciler reconciles a Plotter object
+type PlotterReconciler struct {
+	client.Client
+	Name           string
+	Log            logr.Logger
+	Scheme         *runtime.Scheme
+	ClusterManager multicluster.ClusterManager
+}
+
+// +kubebuilder:rbac:groups=app.m4d.ibm.com,resources=plotters,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=app.m4d.ibm.com,resources=plotters/status,verbs=get;update;patch
+
+// Reconcile receives a Plotter CRD
+//nolint:dupl
```

Comment:
[out of scope feedback] I have some code for a reusable ReconcileCycle that can use instead of duplicating (we can create a variation of it based on our common code). I will open a separate issue.